index.html

```diff
@@ -0,0 +1,186 @@
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/68a31b9037.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <!-- Header with nav bar -->
+    <header class="header">
+      <nav class="nav-container">
+        <div class="name">Smeea</div>
+        <div class="links">
+          <div><a href="#about-me">About me</a></div>
+          <div><a href="#projects">Projects</a></div>
+          <div><a href="#c-me">Contact Me</a></div>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <!--About me-->
+      <section id="about-me" class="section-container">
+        <div class="about-me-image">
+          <img
+            class="profile-image"
+            src="./assets/images/woman.png"
+            alt="avatar of a woman"
+          />
+        </div>
+        <div class="about-me-info">
+          <h1>Smeea</h1>
+          <p>
+            A highly motivated and enthusiastic team player, I am keen to pursue
+            a career as a web developer.
+          </p>
+          <h2>Skills</h2>
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Java Script</li>
+          </ul>
+        </div>
+      </section>
+
+      <!-- Projects -->
+      <section id="projects">
+        <h2 class="project-heading">Projects</h2>
+        <div>
+          <div class="first-project">
```

Comment:
Try to section out your code by leaving spare lines between individual sections. Here it would help us to see clearly where one card starts and the other one ends. Your comments help with this, but the empty lines really help to break the code up even further.